packages/engine-server/src/markdown/remark/dendronPub.ts

```diff
@@ -152,13 +152,16 @@ function plugin(this: Unified.Processor, opts?: PluginOpts): Transformer {
         }
 
         const copts = opts?.wikiLinkOpts;
-        if (note && opts?.transformNoPublish) {
+        if (!note && opts?.transformNoPublish) {
+          value = "403";
+          addError(proc, new DendronError({ message: "no note" }));
```

Comment:
can you add 403 code to the dendron error + severity?

Reply:
see https://github.com/dendronhq/dendron/pull/1344